specs/opcode/3cEXTCODECOPY.md

```diff
@@ -25,7 +25,7 @@ The `EXTCODECOPY` circuit constrains the values popped from stack, call context/
    - `memory_offset` is at the second position of the stack
    - `code_offset` is at the third position of the stack
    - `size` is at the fourth position of the stack
-   - `code_hash` from the address
+   - `code_hash` from the address if witness value `exists == 1`. Otherwise only lookup the account non-existing proof.
```

Comment:
This looks correct to me. We will need to witness/hint the existance of the account even if we do the lookup. But would be nice if someone else could double-check!